lib/feed.ts

```diff
@@ -0,0 +1,21 @@
+import {ErrorCode} from './constants';
+
+export interface BitcoinPriceFeed {
+  getSpotPrice(): Promise<number>;
+}
+
+// https://developers.coinbase.com/docs/wallet/guides/price-data
+export class CoinbaseFeed implements BitcoinPriceFeed {
+  url: string;
+
+  constructor(currency: string) {
+    this.url = `https://api.coinbase.com/v2/prices/spot?currency=${currency}`;
+  }
+
+  async getSpotPrice(): Promise<number> {
```

Comment:
I think we should, in general, stay away from `number` for anything related to money.
`number` uses a floating-point representation so it will in general not be exact.
I think we should agree on a type, probably `Big`, and use this exclusively in the application.

Reply:
Could we use the `MonetaryAmount` for fiat currency as well?